### PR TITLE
change regex for aliases in broken-aliases.xsl

### PR DESCRIPTION
**Pull request by Graur**
#699

### CODE REVIEW

File: eo-parser/src/main/resources/org/eolang/parser/errors/broken-aliases.xsl [hunk 1]
```diff
@@ -28,7 +28,7 @@ SOFTWARE.
     <xsl:copy>
       <xsl:apply-templates select="node()|@*"/>
       <xsl:for-each select="/program/metas/meta[head='alias']">
-        <xsl:if test="not(matches(tail, '^([a-z]+ )?[\w\d]+(\.[\w][\w\d_-]*)*$'))">
+        <xsl:if test="not(matches(tail, '^([\w]+ )?[\w\d]+(\.[\w][\w\d_-]*)*$'))">
```
Comment (yegor256) [inline eo-parser/src/main/resources/org/eolang/parser/errors/broken-aliases.xsl:31]: @Graur this is not entirely correct. `\w` means `[a-zA-Z]`, but identifiers in EO can't start with `[A-Z]`. the first letter should always be small: `[a-z]`

Comment (yegor256) [reply eo-parser/src/main/resources/org/eolang/parser/errors/broken-aliases.xsl:31]: @Graur see https://github.com/objectionary/eo/blob/master/paper/sections/syntax.tex#L20-L27

**Comment by Graur** [comment]
@yegor256 please, have a look

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**